Package.swift

```diff
@@ -54,7 +54,7 @@ automatic linking type with `-auto` suffix appended to product's name.
 let autoProducts = [swiftPMProduct, swiftPMDataModelProduct]
 
 let useSwiftCryptoV2 = ProcessInfo.processInfo.environment["SWIFTPM_USE_SWIFT_CRYPTO_V2"] != nil
```

Comment:
Are there any plans to update `build-script` and the CI to swift-crypto v2? Since this variable is not set, I think [it's still testing against 1.1.5](https://github.com/apple/swift/blob/2ff10a57f40da9a72a897b36fe0e15e533748e08/utils/update_checkout/update-checkout-config.json#L97).

Reply:
> Since this variable is not set, I think it's still testing against 1.1.5

Right, SwiftPM is still using swift-crypto v1.

> Are there any plans to update build-script and the CI to swift-crypto v2?

No specific timeline yet. So far I've been testing locally (to do additional test cases that CI can't cover) whenever there's a new swift-crypto release. When we are ready to upgrade to v2 we'll probably remove this env var and stick with `build-script`.